portfolio/src/main/webapp/index.html

```diff
@@ -22,11 +22,26 @@ <h3>
           </div>
       </h3>
       <iframe src="https://lev-stambler.github.io/Portfolio/"></iframe>
-      <p>Or go to my portfolio site by clicking <a href="https://lev-stambler.github.io/Portfolio/">this link.</a> I made it using a custom frontend framework I created in 2018 named trivial.JS</p>
-
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>Or go to my portfolio site by clicking <a href="https://lev-stambler.github.io/Portfolio/">this link.</a> I made it using a custom frontend framework I created in 2018 named trivial.JS</p>	
+      <br />
+      <form method="POST" action="/data">
+        <input type="text" name="comment" placeholder="Add a comment!" />
+        <input type="checkbox" id="shouldAddComment--checkbox" name="shouldAddComment" value="true" />
+        <label for="shouldAddComment--checkbox">Do you really want to do this?</label>
```

Comment:
FYI, another way to do this that lets you avoid IDs (and visibly groups things a little nicer) is

```
<label>
    <input type="checkbox" name="shouldAddComment" value="true" />
    Do you really want to do this?
</label>